client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -20,6 +20,16 @@ export default function OnboardingEdit() {
 		( e: React.ChangeEvent< HTMLInputElement > ) => setSiteTitle( e.target.value ),
 		[ setSiteTitle ]
 	);
+	const handleSiteTypeChange = useCallback(
+		( e: React.ChangeEvent< HTMLInputElement > ) => setSiteType( e.target.value ),
+		[ setSiteType ]
+	);
+	const siteTypeOptions = [
+		{ label: NO__( 'with a blog.' ), value: SiteType.BLOG },
+		{ label: NO__( 'for a store.' ), value: SiteType.STORE },
+		{ label: NO__( 'to write a story.' ), value: SiteType.STORY },
+	];
+	const handleResetSiteType = useCallback( () => setSiteType( '' ), [ setSiteType ] );
```

Comment:
```suggestion
	const resetSiteType = useCallback( () => setSiteType( '' ), [ setSiteType ] );
```

maybe? See https://github.com/Automattic/wp-calypso/pull/37248/files#r341916152

Reply:
Also, see https://github.com/Automattic/wp-calypso/pull/37248/files#r341918206